app/src/main/java/org/simple/clinic/drugstockreminders/DrugStockReminderApi.kt

```diff
@@ -0,0 +1,13 @@
+package org.simple.clinic.drugstockreminders
+
+import retrofit2.Call
+import retrofit2.http.Body
+import retrofit2.http.GET
+
+interface DrugStockReminderApi {
+
+  @GET("v4/drug_stock")
+  fun drugStockReminder(
+      @Body body: DrugStockReminderRequest
```

Comment:
We use `@Body` annotation to pass objects usually, in this case, we are just passing date in the object. Should we instead use the `@Query` parameter to pass the date in the API function, WDYT?

```kt
@GET("v4/drug_stock")
fun drugStockReminder(
  // Depending on whether retrofit takes in a `LocalDate` as query parameter, we might have to use a `String`
  @Query("date") date: LocalDate/String
): Call<DrugStockReminderResponsePayload>
```

Reply:
Also, it looks like the API is expecting the date as a query parameter. So, passing it as a request body might not work iirc.